### ISSUE TITLE
example sample data (too large/size not warned)

### ISSUE THREAD

**Issue by teuben** [amanchokshi/mwa-satellites#6]
"We now download sample data required to run the following examples"
would be nice to tell the user how many MB typically will be needed.
I found this a bit large, and being at home, took a long time, plus i
just ran out of disk space before it could finish.

**Comment by teuben** [comment]
After - ahum - another two attempts (see my issue on the Sample-Data repo) i noticed it needed 3.8GB. So one should really warn users 4GB is needed.

**Comment by amanchokshi** [comment]
Thanks Peter, I definitely agree that there should be a warning. I've updated the [EMBERS documentation](https://embers.readthedocs.io/en/latest/embersbyexample.html) with an explicit warning. 

When I look in the data directory the actual data is only ~2.5 GB while the .git history has ~1.5GB. I've tried to clone the repo using `git clone --depth 1` to limit the git history to only the last commit, but that didn't reduce the data size. I'm open to any suggestions, but feel like the actual 2.5 GB of data is required.
